actions/websocket_actions.jsx

```diff
@@ -1223,6 +1227,8 @@ function handleChannelViewedEvent(msg) {
 
 export function handlePluginEnabled(msg) {
     const manifest = msg.data.manifest;
+    dispatch({type: ActionTypes.RECEIVED_WEBAPP_PLUGIN, data: manifest});
```

Comment:
Should we trigger a fetch bindings when the status of the apps plugin is changed to enabled?
Should we trigger a binding clear when the status of the apps plugin is changed to disabled?

Reply:
Done :+1: